network/dag/transaction.go

```diff
@@ -77,6 +77,7 @@ type SubscriberTransaction interface {
 	Signable
 	Referenceable
 	PayloadReferencer
+	NetworkHeader
```

Comment:
Thanks for making me feel clairvoyant when we discussed introducing this type :-p